tasks/section5.yml

```diff
@@ -151,15 +188,77 @@
       - rule_5.1.8
 
 - name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
-  command: /bin/true
+  file:
+      dest=/etc/at.deny
+      state=absent
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
+  file:
+      dest=/etc/at.allow
+      state=touch
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md